pkg/kritis/cryptolib/verifier.go

```diff
@@ -37,6 +37,8 @@ type Verifier interface {
 type PublicKey struct {
 	// KeyType stores the type of the public key, one of Pgp, Pkix, or Jwt.
 	KeyType KeyType
+	// Signature Algorithm holds the signing and padding algorithm for the signature.
+	SignatureAlgorithm SignatureAlgorithm
```

Comment:
Should we update [NewPublicKey](https://github.com/grafeas/kritis/blob/5e94b81b2d686303e6cd49653c80c36e290262cf/pkg/kritis/cryptolib/verifier.go#L51) to have a `signatureAlgorithm` argument?

Reply:
done.

Reply:
Actually, changing this causes errors in another file and test other than the verifier. Is it worth pulling this out in another PR?

Reply:
Yeah maybe we should update this afterwards. Could you create an issue for it so we don't forget?